interface/forms/group_attendance/new.php

```diff
@@ -0,0 +1,105 @@
+<?php
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+include_once ("$srcdir/group.inc");
+include_once("statics.php");
+
+$returnurl = 'encounter_top.php';
+
+$participants = getParticipants($therapy_group);
+?>
+<html>
+
+<head>
+<?php html_header_show();?>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/datatables.net-jqui-1-10-11/css/dataTables.jqueryui.min.css" type="text/css">
+<script src="<?php echo $GLOBALS['assets_static_relative'];?>/jquery-min-1-9-1/index.js"></script>
+<script src="<?php echo $GLOBALS['assets_static_relative'];?>/jquery-ui-1-10-4/ui/jquery.ui.core.js"></script>
+<script src="<?php echo $GLOBALS['assets_static_relative'];?>/datatables.net-1-10-11/js/jquery.dataTables.min.js"></script>
+<script src="<?php echo $GLOBALS['web_root'];?>/library/dialog.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot']; ?>/interface/forms/group_attendance/js/functions.js"></script>
+
+
+
+</head>
+
+<body class="body_top">
+
+<form id="group_attendance_form" method=post action="<?php echo $rootdir;?>/forms/group_attendance/save.php?mode=new" name="my_form">
```

Comment:
need onclick='top.restoreSession()' in here. See here for details:
http://www.open-emr.org/wiki/index.php/OpenEMR_System_Architecture#PHP_Sessions_and_Browser_Windows